packages/mongodb/PKGBUILD

```diff
@@ -1,67 +1,91 @@
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Thomas Dziedzic < gostrc at gmail >
+# Contributor: Mathias Stearn <mathias@10gen.com>
+# Contributor: Alec Thomas
+# Contributor: Fredy García <frealgagu at gmail dot com>
 
 pkgname=mongodb
+_pkgname=mongodb
 # #.<odd number>.# releases are unstable development/testing
-pkgver=5.0.7
+pkgver=6.0.2
 pkgrel=1
-pkgdesc='A high-performance, open source, schema-free document-oriented database.'
-arch=('x86_64' 'aarch64')
-url='https://www.mongodb.com/try/download/community'
-license=('custom:SSPL')
-depends=('curl' 'libstemmer' 'snappy' 'gperftools' 'boost-libs' 'pcre'
-         'yaml-cpp')
-makedepends=('scons' 'python-psutil' 'python-setuptools' 'python-regex'
-             'python-cheetah3' 'python-yaml' 'python-requests' 'boost')
-optdepends=('mongodb-tools: mongoimport, mongodump, mongotop, etc')
+pkgdesc="A high-performance, open source, schema-free document-oriented database"
+arch=('x86_64')
+url="https://www.mongodb.com/"
+license=("Apache" "custom:SSPL1")
+depends=('libstemmer' 'snappy' 'boost-libs' 'pcre' 'yaml-cpp' 'curl')
+makedepends=('python-psutil' 'python-setuptools' 'python-regex' 'python-cheetah3' 'python-yaml' 'python-requests' 'python-pymongo' 'boost')
+optdepends=('mongodb-tools: mongoimport, mongodump, mongotop, etc'
+            'mongosh-bin: interactive shell to connect with MongoDB')
 backup=("etc/mongodb.conf")
-source=("https://fastdl.mongodb.org/src/$pkgname-src-r$pkgver.tar.gz"
-        'mongodb.sysusers'
-        'mongodb.tmpfiles'
-        'mongodb-5.0.2-skip-no-exceptions.patch'
-        'mongodb-5.0.2-no-compass.patch'
-        'mongodb-4.4.1-boost.patch')
-sha512sums=('cc8d65973e4204724ce2e6da7edb9f9922dd6e0e686904dcce715d93065a7f19521f5790c9fb86b78476a0425dad47155257f93238cd26db7abd395daf4bd410'
-            '36a2c4336d6aeeec303dd0aec2c554db826d479efcb95269f75e2150ad752516c3f25c50f8e25b90384dea65b5b5b72a15ab17d5da35f6f92f20cc65af02f61b'
-            '41b711a623b034b66e146487fab948608111af48aff2b4f1b171b64b531fa6a1b36de8b348ff543fdd9c3ac34a1a20fae9b23cdb124abda93f89ce011a7921ef'
-            '6750b5d8427616b290d285959c0c43a28409349ae7b52153d524590a8a38c26c283acc25b82410853b5c87a4c74927cf5844580778d1ef4378cd1b035844dc97'
-            '07fe714a4256c13449e1e764c88d5051773aa9397228085a8d581cde91477dff2c3dc70ba801eb559af76ff19abf205d32c9510606a9bb26cebb33f68822e607'
-            '504ec9a4e41f5e1c1a069d64e0958b260a3a8506d17638e62ddf6f1cd873b835f33a588b779e4851d63351faef4be1e0f4720e86f389128d062909b58fdb9aa2')
+source=(https://fastdl.mongodb.org/src/mongodb-src-r$pkgver.tar.gz
+        mongodb.sysusers
+        mongodb.tmpfiles
+        mongodb-4.4.1-gcc11.patch
+        mongodb-5.0.2-no-compass.patch
+        mongodb-5.0.2-skip-reqs-check.patch
+        mongodb-6.0.0-boost-1.79.patch
+        mongodb-6.0.2-sconstruct-changes.patch
+        mongodb-6.0.0-gcc12.patch)
+sha256sums=('39fa695a3a79008c818cb92df1fa716510d66069f7cec20a8fb3051286ac34bf'
```

Comment:
all BA PKGBUILD use `sha512sums`

Reply:
Changed to sha512sums and version bumped to the latest one 6.0.4.